src/Resources/contao/dca/tl_module.php

```diff
@@ -66,9 +68,11 @@
 $GLOBALS['TL_DCA']['tl_module']['fields']['mailchimpMandatoryInterests'] = [
     'label' => &$GLOBALS['TL_LANG']['tl_module']['mailchimpMandatoryInterests'],
     'inputType' => 'checkbox',
+    'options_callback' => ['Oneup\Contao\MailChimpBundle\EventListener\DcaListener', 'onLoadInterests'],
```

Comment:
Is it ok to use `Oneup\Contao\MailChimpBundle\EventListener\DcaListener::class`?

Reply:
Yeah, that's the way to go, didn't think of that.

Reply:
Ping @fritzmg who's gonna change? you or me?

Reply:
done